<ui version="4.0" >
 <class>UiMSSqlConnectionBase</class>
 <widget class="QWidget" name="UiMSSqlConnectionBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>245</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>409</width>
    <height>245</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>409</width>
    <height>245</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>MS SQL Connection Assistant</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="minimumSize" >
      <size>
       <width>409</width>
       <height>69</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>409</width>
       <height>69</height>
      </size>
     </property>
     <property name="pixmap" >
      <pixmap resource="../../resource/dms.qrc" >:/banner/images/mssql2005_banner.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="Line" name="line_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1" >
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="labelHost" >
           <property name="minimumSize" >
            <size>
             <width>110</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>110</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text" >
            <string>Server:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditoHost" />
         </item>
        </layout>
       </item>
       <item row="1" column="0" >
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="labelUser" >
           <property name="minimumSize" >
            <size>
             <width>110</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>110</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text" >
            <string>Username:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditUser" />
         </item>
        </layout>
       </item>
       <item row="2" column="0" >
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="labelPassword" >
           <property name="minimumSize" >
            <size>
             <width>110</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>110</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text" >
            <string>Password:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditPassword" >
           <property name="echoMode" >
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnCheckConnection" >
           <property name="minimumSize" >
            <size>
             <width>26</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>26</width>
             <height>26</height>
            </size>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset resource="../../resource/dms.qrc" >:/picture/16/images/16x16/reload_16.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0" >
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="labelDatabase" >
           <property name="minimumSize" >
            <size>
             <width>110</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>110</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text" >
            <string>Database:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxDatabase" />
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="2" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0" >
    <widget class="Line" name="line_3" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>5</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QPushButton" name="btnOk" >
         <property name="minimumSize" >
          <size>
           <width>110</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>110</width>
           <height>26</height>
          </size>
         </property>
         <property name="text" >
          <string>&amp;Ok</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancel" >
         <property name="minimumSize" >
          <size>
           <width>110</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>110</width>
           <height>26</height>
          </size>
         </property>
         <property name="text" >
          <string>&amp;Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>5</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resource/dms.qrc" />
 </resources>
 <connections/>
</ui>
